canbench-bin/src/print_benchmark.rs

```diff
@@ -1,11 +1,13 @@
 use canbench_rs::{BenchResult, Measurement};
 use colored::Colorize;
 
-// The threshold that determines whether or not a change is significant.
-const NOISE_THRESHOLD: f64 = 2.0;
```

Comment:
No change.

There was no explanation here why 2.0 is the one size that fits all. Therefore, I think it is very reasonable to assume that there is nothing special about the number 2.0 such that everyone should use it.

In general, when constants like this have no explanation for how their value was chosen, it means that an explanation is missing, or it should not be a constant in the first place, and should instead be specified by callers. It is reasonable for the tool to have a default for convenience (which is what you have done here).